frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -53,6 +84,23 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     InfrastructureModel,
     'cluster',
   );
+  const [ocsSubscription, ocsSubscriptionLoaded, ocsSubscriptionError] = useK8sWatchResource<
+    SubscriptionKind
+  >(ocsSubscriptionResource);
+  const [packageManifest, packageManifestLoaded, packageManifestError] = useK8sWatchResource<
+    PackageManifestKind
+  >(PackageManifestResource);
+  const memoizedPackageManifest = useDeepCompareMemoize(packageManifest, true);
+  const memoizedOcsSubscription = useDeepCompareMemoize(ocsSubscription, true);
+  const currentChannel =
+    memoizedOcsSubscription?.spec?.channel ?? memoizedPackageManifest?.status?.channels?.[0]?.name;
+  const filteredVersions = memoizedPackageManifest?.status?.channels?.filter(
+    (channel) =>
+      parseFloat(channel.name.substring(channel.name.indexOf('-') + 1)) >
+      parseFloat(currentChannel.substring(currentChannel.indexOf('-') + 1)),
```

Comment:
```suggestion
      currentChannelVersion,
```
Create a var that is currentChannelVersion no need to compute it everytime in a loop.